<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectPage</class>
 <widget class="QWidget" name="ConnectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>629</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ConnectPage {
    background-color: #ffffff;
	font-family: &quot;Ubuntu&quot;, monospace;
}

QLabel#titleLabel {
    font-size: 60px;
    font-weight: 900;
    padding: 30px;
    letter-spacing: 8px;
    color: #800000;
    background: transparent;
}

QLabel#subtitleLabel {
    font-size: 14px;
    padding-bottom: 30px;
    letter-spacing: 3px;
    color: #000000;
    background: transparent;
}

QGroupBox#connectGroup {
    background: qlineargradient(x1:0,y1:0,x2:0,y2:1,
        stop:0 #f2f2f2, stop:1 #d9d9d9);
    border: 2px solid #9e9e9e;
    border-radius: 4px;
    margin-top: 30px;          
    padding: 40px 20px 20px 20px; 
    font-size: 16px;
    font-weight: 900;
    color: #000000;
    letter-spacing: 2px;
}

QGroupBox#connectGroup::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 10px;
    color: #000000; 
    font-size: 18px;
    font-weight: 900;
    letter-spacing: 2px;
}

QLineEdit#hostEdit,
QLineEdit#portEdit {
    background: #f5f5f5;
    border: 1px solid #cfcfcf;
    border-radius: 4px;
    padding: 8px 12px;
    color: #000000;
    font-size: 14px;
    selection-background-color: #cfcfcf;
    selection-color: #000000;
}

QLineEdit#hostEdit:focus,
QLineEdit#portEdit:focus {
    border-color: #7a7a7a;
}

QLabel#hostLabel,
QLabel#portLabel {
    color: #000000; 
    font-size: 14px;
    font-weight: bold;
    background: transparent;
}

QPushButton#connectButton {
    background: qlineargradient(x1:0,y1:0,x2:0,y2:1,
        stop:0 #f2f2f2, stop:1 #d9d9d9);
    border: 2px solid #9e9e9e;
    border-radius: 4px;
    padding: 20px 50px;
    font-size: 16px;
    font-weight: 900;
    color: #000000;
    letter-spacing: 3px;
}

QPushButton#connectButton:hover {
    background: qlineargradient(x1:0,y1:0,x2:0,y2:1,
        stop:0 #ffffff, stop:1 #e6e6e6);
    border-color: #7a7a7a;
}

QPushButton#connectButton:pressed {
    background: qlineargradient(x1:0,y1:0,x2:0,y2:1,
        stop:0 #cfcfcf, stop:1 #bdbdbd);
}

QLabel#statusLabel {
    font-size: 12px;
    padding: 20px;
    letter-spacing: 2px;
    color: #4facfe; 
    background: transparent;
}

QLabel#statusLabel[state=&quot;ok&quot;] {
    color: #00c853; 
}

QLabel#statusLabel[state=&quot;error&quot;] {
    color: #ff6b6b; 
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="spacerTop">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>-1</pointsize>
       <fontweight>Black</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>NEOBANK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>-1</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>&gt; Требуется подключение &lt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="connectGroup">
     <property name="title">
      <string>[ Параметры подключения ]</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Хост:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hostEdit">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Порт:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="portEdit">
        <property name="inputMask">
         <string>00000;_</string>
        </property>
        <property name="text">
         <string>4433</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="connectButton">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>-1</pointsize>
       <fontweight>Black</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Подключиться</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>-1</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>&gt; Не подключено &lt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacerBottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
